suite-common/message-system/src/messageSystemTypes.ts

```diff
@@ -83,6 +83,9 @@ const getTradingContext = (type: TradingType) => `trading.${type}` as const;
 export type SettingsCategory = 'general' | 'device' | 'networks' | 'debug';
 const getSettingsContext = (category: SettingsCategory) => `settings.${category}` as const;
 
+export type LegalContextKey = 'gateway';
+const getLegalContext = (key: LegalContextKey) => `legal.${key}` as const;
```

Comment:
```suggestion
const getLegalContext = (key: LegalContextKey) => `legal.${key}` as const;
```

How about `staking.footer.legal` and `trading.footer.legal` or something like that. I dont like calling it `legal` 😀

Reply:
I had a long debate about this with @jbazant because I prefer semantic naming instead of UI (what if we move the message somewhere else in the future?). I think `legal` works because it describes the root reason, not just the placement, but if you don't like that word, what about `compliance` or `regulation`? 😄

Reply:
okay 😀 let's keep it